documentation/README.md

```diff
@@ -0,0 +1,23 @@
+# Boundary Developer conventions
+
+## I18n:
+
+We use [ember-intl](https://ember-intl.github.io/ember-intl/) to manage i18n. All the code related to i18n lives within [the core addon](https://github.com/hashicorp/boundary-ui/tree/main/addons/core).
+
+All the features provided by ember-intl are available to use, on top of that, the Boundary team [provides some custom helpers](https://github.com/hashicorp/boundary-ui/tree/main/addons/core/addon/helpers).
+
+### Dates & Time
+
+Dates and time must be expressed when possible using the [`<time>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).
+
+The desired date format for the Boundary teams is: `YYYY-MM-DD HH-MM-SS TZ`
+
+### Strings
+
+Plain text strings must be avoided, we use a rules [(`no-bare-strings`)](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-bare-strings.md) within our linter to avoid committing plain text strings.
```

Comment:
```suggestion
Plain text strings must be avoided, we use a rule [(`no-bare-strings`)](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-bare-strings.md) within our linter to avoid committing plain text strings.
```

Reply:
@lisbet-alvarez Nice catch!!